nova/templates/etc/_api_audit_map.conf.tpl

```diff
@@ -0,0 +1,72 @@
+[DEFAULT]
+# default target endpoint type
+# should match the endpoint type defined in service catalog
+target_endpoint_type = None
```

Comment:
Does this value need changing from `None` to the actual service catalog endpoint type?